src/app/chat/event_handling.nim

```diff
@@ -29,6 +31,14 @@ proc handleChatEvents(self: ChatController) =
     let notifications = ActivityCenterNotificationsArgs(e).activityCenterNotifications
     self.view.pushActivityCenterNotifications(notifications)
 
+  self.status.events.on("new-contactNicknameChanged") do(e: Args):
+    var evArgs = ContactNicknameUpdatedArgs(e)
+    #let contact = Profile(
+    #  id: evArgs.contactId,
+    #  localNickname: evArgs.nickname
+    #)
+    #self.view.updateChannelForContacts(@[contact])
```

Comment:
It seems the change in local nicknames via profile popup isn't propagated to the view because the code uses the new `app_service` which emits `new-contactNicknameChanged`, which this module here isn't aware of.

So what I'm trying to do is to have it listen to that event so we can update the view accordingly (similar to what it does in `contactUpdate` below).

However, adding line 35 causes the app to crash. 

```
Object 0x7ff881013a80 destroyed while one of its QML signal handlers is in progress.
Most likely the object was deleted synchronously (use QObject::deleteLater() instead), or the application is running a nested event loop.
This behavior is NOT supported!
qrc:/app/AppLayouts/Chat/popups/ProfilePopup.qml:244: function() { [native code] }
```

The `ContactNicknameUpdatedArgs` type has been introduced in `status-lib` in a branch with the same name as this one. This commit is also pointing at that branch.

So the type exists and the code is compiling just fine.